packages/messaging/lib/messages/DlcAccept.ts

```diff
@@ -126,6 +127,51 @@ export class DlcAcceptV0 extends DlcAccept implements IDlcMessage {
     };
   }
 
+  /**
+   * Validates correctness of all fields in DlcOffer
+   * https://github.com/discreetlogcontracts/dlcspecs/blob/master/Protocol.md#the-offer_dlc-message
+   * @throws Will throw an error if validation fails
+   */
+  public validate(): void {
```

Comment:
Would be great to have some validation for these checks

- if any input in funding_inputs is not a BIP141 (Segregated Witness) input.
  - MUST reject the contract.

Reply:
Hey good point, can you take a look at the updated changes again?